sos/policies/runtimes/__init__.py

```diff
@@ -147,6 +147,23 @@ def get_volumes(self):
                     vols.append(ent[-1])
         return vols
 
+    def container_exists(self, container):
+        """Check if a given container ID or name exists on the system from the
+        perspective of the container runtime.
+
+        Note that this will only check _running_ containers
+
+        :param container:       The name or ID of the container
+        :type container:        ``str``
+
+        :returns:               True if the container exists, else False
+        :rtype:                 ``bool``
+        """
+        for _contup in self.containers:
+            if container in _contup:
```

Comment:
When testing this with `podman`, the `_contup` is a pair of container ID and image name, not the container name, like:

    # podman ps
CONTAINER ID  IMAGE                                       COMMAND    CREATED        STATUS            PORTS  NAMES
    902ab039c904  registry.access.redhat.com/ubi8/ubi:latest  /bin/bash  4 minutes ago  Up 4 minutes ago         nice_einstein
    e214612764d3  registry.access.redhat.com/ubi8/ubi:latest  /bin/bash  4 minutes ago  Up 4 minutes ago         xenodochial_bohr
    # 

While printing the pair gets me:

    902ab039c904 registry.access.redhat.com/ubi8/ubi:latest
    e214612764d3 registry.access.redhat.com/ubi8/ubi:latest

Did I something wrong or ar we really storing image name instead of container name? See https://github.com/sosreport/sos/blob/main/sos/policies/runtimes/__init__.py#L89-L92 how we feed the `self.containers` list.

Reply:
I also tested and you are right about _contup is a tuple (ID, container name). 

[self.containers ](https://github.com/sosreport/sos/blob/main/sos/policies/runtimes/__init__.py#L89-L92 ) are ok as ([0],**[-1]**) so we are storing ID and container name.

Reply:
The name should always be the last field for podman:

```
$ podman ps
CONTAINER ID  IMAGE       COMMAND     CREATED     STATUS      PORTS       NAMES
```

I've run a few more sanity tests locally, and from that I'm always seeing the name in the last field.

What version of podman are you testing with?

Reply:
Ah I apologize. Typo in my debug (`1` instead of `-1`, hence image name instead of container name). This works well.